dockerfiles/README.md

```diff
@@ -16,7 +16,7 @@ To launch the complete harmoni dev setup in docker:
     For example, to use Wave2Letter (requires running get_w2l_models.sh first):
     - From harmoni_core, open two terminals and:
         - ```roscore```
-        - ```roslaunch harmoni_decision services.launch use_pc_services:=false```
+        - ```roslaunch harmoni_decision services.launch use_harmoni_services:=false```
         - ```roslaunch harmoni_decision routers.launch```
```

Comment:
This README should be updated. To run wave2letter, you do not need anymore to follow those steps. It would be enough to run on the ros_w2l container: 
`rlspeech`